qutip/visualization.py

```diff
@@ -679,12 +720,10 @@ def matrix_histogram(M, xlabels=None, ylabels=None, title=None, limits=None,
     """
 
     # default options
-    default_opts = {'figsize': None, 'cmap': 'jet', 'cmap_min': 0.,
-                    'cmap_max': 1., 'zticks': None, 'bars_spacing': 0.2,
+    default_opts = {'zticks': None, 'bars_spacing': 0.2,
```

Comment:
Could you rename this variable, it is used later with values no longer the default.

Reply:
Hi. I changed the code to use ```options``` argument instead of default_opts and changed the default value from ```None``` to ```{}```.